Gruntfile.js

```diff
@@ -382,6 +386,13 @@ module.exports = function (grunt) {
             config.options.appInsightsInstrumentationKey = grunt.option(telemetryKeyIdentifier);
         }
 
+        // Add unifiedAppVersion value for electron-based products
+        if (config.options.productCategory === 'electron') {
```

Comment:
flagging to say we might want to change the product category to 'unified' from 'electron' to be consistent (not part of this PR)